cli/tools/run/mod.rs

```diff
@@ -77,9 +77,14 @@ pub async fn run_script(
   let worker_factory = factory.create_cli_main_worker_factory().await?;
   let mut worker = worker_factory
     .create_main_worker(mode, main_module.clone())
-    .await?;
-
-  let exit_code = worker.run().await?;
+    .await
+    .inspect_err(|e| {
+      deno_telemetry::report_event("deno_boot_failure", e)
```

Comment:
```suggestion
      deno_telemetry::report_event("boot_failure", e)
```

Reply:
this could conflict with user code emitting events